x-pack/legacy/plugins/monitoring/public/components/chart/horizontal_legend.js

```diff
@@ -61,38 +57,37 @@ export class HorizontalLegend extends React.Component {
   }
 
   createSeries(row, rowIdx) {
-    const classes = ['col-md-4 col-xs-6 monRhythmChart__legendItem'];
+    const classes = ['monRhythmChart__legendItem'];
```

Comment:
I don't think you need this anymore now that there is only one item.

Reply:
We still need this array because we're adding things to it with the `if`s below.